docs/utilities/idempotency.md

```diff
@@ -259,6 +259,20 @@ payload validation, we would have returned the same result as we did for the ini
 returning an amount in the response, this could be quite confusing for the client. By using payload validation on the
 amount field, we prevent this potentially confusing behaviour and instead raise an Exception.
 
+### Making idempotency key required
+
+By default, events without any idempotency key don't raise any exception and just trigger a warning.
+If you want to ensure that at an idempotency is found, you can pass in `raise_on_no_idempotency_key` as True and an
```

Comment:
Not sure what you mean by "If you want to ensure that at an idempotency is found" - could you simplify this to non-native English speakers, please?